compiler/ast/Cargo.toml

```diff
@@ -11,10 +11,8 @@ license = "MIT"
 default = ["constant-optimization", "fold"]
 constant-optimization = ["fold"]
 fold = []
-unparse = ["rustpython-common"]
 
 [dependencies]
-rustpython-compiler-core = { path = "../core", version = "0.2.0" }
-rustpython-common = { path = "../../common", version = "0.2.0", optional = true }
+ruff_text_size =  { path = "../../ruff_text_size" }
```

Comment:
I decided to move `ruff_text_size` into this repository because:

* I want to avoid git submodules in our main repository
* `ruff_text_size` should rarely change